packages/geb/src/proxy-action.ts

```diff
@@ -266,16 +266,16 @@ export class GebProxyActions {
     }
 
     freeTokenCollateral(
-        collateralJoin: string,
         safe: BigNumberish,
-        amt: BigNumberish
+        collateralType: string,
+        wad: BigNumberish
     ): TransactionRequest {
         return this.getProxiedTransactionRequest(
             this.proxyActionCore.freeTokenCollateral(
                 this.addressList.SAFE_MANAGER,
-                collateralJoin,
+                this.collateralToJoin[collateralType],
                 safe,
-                amt
+                wad
```

Comment:
In this file we have `wad`, `collateralAmount` and `collateralValue` all as names for the amount of collateral.
Can we pick one or is there a reason for the differences?

Reply:
We can pick one. The thing needs a cleanup indeed, but since all functions are slightly different I followed the same names/variables as the ETH calls. I'll fix this tomorrow.